Formula/gambit-scheme.rb

```diff
@@ -0,0 +1,127 @@
+class GambitScheme < Formula
+  desc "Gambit Scheme"
+  homepage "http://gambitscheme.org"
+  url "https://github.com/gambit/gambit.git", :tag => "v4.9.0"
```

Comment:
Needs associated `:revision`. And we prefer to build from github tarball if possible.